op.yml

```diff
@@ -23,6 +23,12 @@ inputs:
                   format: base64
           credsStore:
             type: string
+  jqRef:
```

Comment:
What's the problem we're trying to solve by making image configurable?

Reply:
Not having to rev this package ever again.

Reply:
Just my personal opinion but the encapsulation is nice, either the op works or it doesn't, if it doesn't it should be fixed. If you're strongly opinionated though I'll approve it.